### PR TITLE
feature: add peft baseline benchmark

### PR DESCRIPTION
**Pull request by yezhem**

### CODE REVIEW

File: mlora/model/model_llama.py
Comment (merlintang) [inline mlora/model/model_llama.py:1]: why we need to change this part to support PEFT

Comment (yezhem) [reply mlora/model/model_llama.py:1]: this part does not used by PEFT (the new file peft_baseline.py is just a baseline),  nvidia nsight-system use this part, we can use:  `nsys stats --report` to generate the time cost.

File: mlora/model/LoraLiner.py [hunk 1]
```diff
@@ -104,11 +104,13 @@ def replace_init_lora_tensor(lora: Lora, lora_a: torch.Tensor, lora_b: torch.Ten
     def forward(self, data: torch.Tensor, input_args: MultiLoraBatchData) -> torch.Tensor:
         # data shape is: batch_size * max_seq_len * dim
         # result = data @ self.weight_.transpose(0, 1)
-        result = self.weight_.forward(data)
+        with torch.cuda.nvtx.range("liner_base"):
```
Comment (Antlera) [inline mlora/model/LoraLiner.py:107]: "liner_base" -> "linear_base"?

File: mlora/model/LoraLiner.py [hunk 2]
```diff
@@ -104,11 +104,13 @@ def replace_init_lora_tensor(lora: Lora, lora_a: torch.Tensor, lora_b: torch.Ten
...
+            result = self.weight_.forward(data)
 
         if not self.enable_lora_:
             return result
 
+        torch.cuda.nvtx.range_push("linear_lora")
```
Comment (Antlera) [inline mlora/model/LoraLiner.py:113]: Better to profile each LoRA adapter separately?

Comment (yezhem) [reply mlora/model/LoraLiner.py:113]: good idea, i will modify it

**Comment by merlintang** [comment]
LGTM

**Pull request closed without merging**